pkg/ocm/versions.go

```diff
@@ -248,9 +248,13 @@ func IsValidVersion(userRequestedVersion string, supportedVersion string, cluste
 	if err != nil {
 		return false, err
 	}
+
+	isPreRelease := a.Prerelease() != "" && b.Prerelease() != ""
+
 	versionSplit := a.Segments64()
-	//If user has specified patch we check directly and return the result
-	if len(versionSplit) > 2 && versionSplit[2] > 0 {
+	// If user has specified patch or not specified patch but is a preRelease
+	// Check directly
+	if len(versionSplit) > 2 && versionSplit[2] > 0 || (versionSplit[2] == 0 && isPreRelease) {
```

Comment:
If it was a preRelease but patch was not 0 it was already taking into account. But if it is a pre release and patch is 0 we still need to check directly